robot/Cumulus/resources/NPSP.robot

```diff
@@ -183,7 +183,28 @@ Run Donations Batch Process
     Wait For Batch To Process    RLLP_OppContactRollup_BATCH        Completed
     Wait For Batch To Process    RLLP_OppHouseholdRollup_BATCH      Completed
     Wait For Batch To Process    RLLP_OppSoftCreditRollup_BATCH     Completed
-    
+
+Validate Batch Process When CRLP Checked
+    Open NPSP Settings          Bulk Data Processes                Rollup Donations Batch
+    Click Settings Button       idPanelOppBatch                    Run Batch
+
+    Wait For Batch To Process    CRLP_Account_SoftCredit_BATCH            Completed
+    Wait For Batch To Process    CRLP_RD_BATCH                            Completed
+    Wait For Batch To Process    CRLP_Account_AccSoftCredit_BATCH         Completed
+    Wait For Batch To Process    CRLP_Contact_SoftCredit_BATCH            Completed
+    Wait For Batch To Process    CRLP_Account_BATCH                       Completed
+    Wait For Batch To Process    CRLP_Contact_BATCH                       Completed
+
+Run Donations Batch Process
+    Go To Page                   Custom                         NPSP_Settings
+    Open Main Menu               Donations
+    Click Link With Text         Customizable Rollups
```

Comment:
Just curious, did open NPSP settings keyword didn't work here

Reply:
I will try it out and update